Lib/pydoc.py

```diff
@@ -2247,8 +2247,8 @@ def log_message(self, *args):
 
     class DocServer(http.server.HTTPServer):
 
-        def __init__(self, port, callback):
-            self.host = 'localhost'
+        def __init__(self, host, port, callback):
+            self.host = host
```

Comment:
@merwok,  I considered that but figured it was a class inside a underscored method so its not guaranteed to remain compatible.  I figured it would be better to opt for the more readable signature instead of being backwards compatible.  If it's a blocker to make this compatible let me know but if we're implying that this function should be treated as if it is public we may want to remove the prefixed underscore but that will definitely break people.